boot-microservices/src/main/java/org/ow2/proactive/boot/microservices/iam/util/OperatingSystem.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive.boot.microservices.iam.util;
+
+/**
+ * OperatingSystem
+ *
+ * @author The ProActive Team
+ */
+public enum OperatingSystem {
```

Comment:
Instead of rewriting this class in your project, I think it would be cleaner to move the OS & OSF classes from common-client to common-api.

I suppose your project could inherit from common-api without much trouble, isn't it ?

Reply:
OS & OSF centralized in 'common-api'. Thanks